config/locales/mailers/fr.yml

```diff
@@ -16,7 +16,7 @@ fr:
       account_deleted: "%{community} - Confirmation de la suppression du compte"
       deletion_requested: "%{community} - Suppression du compte demandée"
       trusted: Félicitations ! Vous êtes maintenant un utilisateur "de confiance" sur %{community} !
-      moderator: 'Félicitations ! Vous êtes le modérateur de #%{tag_name}'
+      moderator: 'Congrats! You''re now a moderator for #%{tag_name}'
```

Comment:
I have two thoughts here. First, I think the yml syntax allows for a syntax like on the prior line, without quote marks wrapping the whole phrase. Since they're single quotes, removing them makes the syntax look less weird when the extra apostrophe in `You''re` is removed too.

Second, these are all changes I'd recommend for the english version of this yml, I notice this change is in the french language version. 

I think the best overall approach might be to 
- set the en.yml version as `Congrats! You're now a moderator for #%{tag_name}` without wrapping quotes.
- keep the French language in place, with or without removing the wrapping quote marks (standardizing the differing styles around wrapping quotes and interpolating text in these ymls seems outside the scope of this helpful PR)

Reply:
Thank you for your comment. Yes, obviously better not to have single quotes. Since `#` in both languages prevents quotation-less format, is it fine to wrap them with double quotes?

Reply:
Yes, I think that would work and fit in with other usages nearby in the file. Thank you for this context!